src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -388,14 +388,31 @@ public void clearEntries() {
       update();
     }
 
+    @Nullable
+    private MutableTreeNode parentStackTraceNode = null;
+
     public void appendNodes(List<FlutterLogEntry> entries) {
       if (treeTable == null || uiThreadAlarm.isDisposed()) {
         return;
       }
-
       uiThreadAlarm.addRequest(() -> {
         final MutableTreeNode root = getRoot();
-        entries.forEach(entry -> insertNodeInto(new FlutterEventNode(entry), root, root.getChildCount()));
+
+        entries.forEach(entry -> {
+          final FlutterEventNode newNode = new FlutterEventNode(entry);
+          // TODO: How do we enable collapse ????
```

Comment:
@pq Do you have any idea to make FlutterLogTree can collapse ?

Reply:
Hmmm.  I don't really know the right way to do it though I notice that `JTree`s have a `collapsePath` method.  So maybe something like:

      treeTable.getTree().collapsePath(...);

?

Of course it would be nicer if this was done in the model.  Maybe there's something I'm not seeing?  I'll do a little more looking.